index.md

```diff
@@ -1,3 +1,4 @@
+<div class='oldSection'>
```

Comment:
everything inside this div should be indented one level deeper, right?

Reply:
mm one level deeper from where? Anyway, I think this oldSection will disappear, I added it just to show while we are doing the rest of the page.